packages/typespec-ts/test/modularIntegration/azureExampleClient.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { AzureExampleClient } from "./generated/azure/example/basic/src/index.js";
+import { assert } from "chai";
+describe("AzureExampleClient Client", () => {
+  let client: AzureExampleClient;
+
+  beforeEach(() => {
+    client = new AzureExampleClient({
+      allowInsecureConnection: true,
+      endpoint: "http://localhost:3002"
+    });
+  });
+
+  it("basic example client action", async () => {
```

Comment:
I prefer to keep this sample file considering there is only one and there is no index file to track their change.

Reply:
is this not a test file? I don't see any sample file being kept/generated for this cadl ranch test?

Reply:
wired it should be there, @v-jiaodi why this file is not updated and there is no ci diff?

Reply:
considering the change is a little bit big(see [pr](https://github.com/MaryGao/autorest.typescript/pull/39/files#diff-a08bbc9524d7e9443058e88120c1fc0867fd19b2521a73e074153fa222f0a58e)) i think i will not track this and in case you are interested in the generated sample is [here](https://github.com/MaryGao/autorest.typescript/blob/fe54165786ac21efc61220146813818520f2b89b/packages/typespec-ts/test/modularIntegration/generated/azure/example/basic/samples-dev/basicActionSample.ts).